modules/ask_va_api/app/lib/ask_va_api/inquiries/payload_builder/inquiry_payload.rb

```diff
@@ -44,7 +46,7 @@ def additional_payload_fields
             DependantRelationship: translate_field(:dependent_relationship),
             InquiryAbout: translate_field(:inquiry_about),
             InquiryCategory: inquiry_params[:category_id],
-            InquirySource: '722_310_000',
+            InquirySource: '722310000',
```

Comment:
Did you consider using the constant?
```suggestion
            InquirySource: UNAUTHENTICATE_ID,
```

Reply:
Good catch. Thank you and updated